shell/browser/native_window_mac.mm

```diff
@@ -509,6 +511,45 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
     [NSEvent removeMonitor:wheel_event_monitor_];
 }
 
+void NativeWindowMac::RepositionTrafficLights() {
+  if (!traffic_light_offsetX_ && !traffic_light_offsetY_) {
```

Comment:
This does not allow offsets set to `0` which someone may want

Reply:
My thinking was that an offset of 0 should result in the default position of the traffic lights (in `titleBarStyle: 'hidden'` mode). Are you suggesting there's a use case where someone might want an offset smaller than that default position? In that case, negative numbers would work here. 

IMO (x: 0, y: 0) should have a frame of reference relative to the default position, and not the absolute (0, 0) position of the frame